Makefile

```diff
@@ -100,3 +100,30 @@ upgrade: ## update the pip requirements files to use the latest releases satisfy
 	grep -e "^django==" requirements/edx/base.txt > requirements/edx/django.txt
 	sed '/^[dD]jango==/d' requirements/edx/testing.txt > requirements/edx/testing.tmp
 	mv requirements/edx/testing.tmp requirements/edx/testing.txt
+
+split_requirements:
+	grep -E    "^-e (common|openedx|lms|cms|\.)" requirements/edx/base.txt > requirements/edx/base_in_tree.txt
+	grep -E -v "^-e (common|openedx|lms|cms|\.)(/| )" requirements/edx/base.txt > requirements/edx/base_not_in_tree.txt
+
+# These make targets currently only build LMS images.
+docker_build:
+	docker build . -f Dockerfile --target lms -t openedx/edx-platform
+	docker build . -f Dockerfile --target lms-newrelic -t openedx/edx-platform:latest-newrelic
+	docker build . -f Dockerfile --target lms-devstack -t openedx/edx-platform:latest-devstack
```

Comment:
This tagging scheme excludes mention of LMS. How about:
* `openedx/lms`
* `openedx/lms-dev`
* `openedx/lms-newrelic`
* `openedx/studio`
* `openedx/studio-dev`
* `openedx/studio-newrelic`

Reply:
Moving this comment over to #24666